core/templates/components/summary-tile/collection-summary-tile.directive.html

```diff
@@ -42,6 +41,15 @@ <h3 class="activity-title protractor-test-collection-summary-tile-title" ng-styl
 </md-card>
 
 <style>
+  .oppia-activity-summary-tile {
```

Comment:
I don't think you can do it like this, because other files also use `oppia-activity-summary-tile`. Please pick a prefix that is specific to classes in this file only.